packages/web3-common/src/web3_base_wallet.ts

```diff
@@ -22,7 +22,7 @@ export abstract class Web3BaseWallet<T extends Web3BaseWalletAccount> {
 		this._accountProvider = accountProvider;
 	}
 
-	public abstract create(numberOfAccounts: number, entropy: string): this;
```

Comment:
Removed entropy as our account creation does not support this, https://github.com/ChainSafe/web3.js/blob/2ea05a8e9b251f618bdc54f0319f0c45896822af/packages/web3-eth-accounts/src/account.ts#L63 its breaking change